archive-map.json

```diff
@@ -1,20 +1,21 @@
 {
   "$id": "https://sage-bionetworks.github.io/mobile-client-json/archive-map.json",
-  "$schema": "https://sage-bionetworks.github.io/mobile-client-json/schemas/v1/ArchiveMap.json",
-  "allOf": [
+  "$schema": "https://sage-bionetworks.github.io/mobile-client-json/schemas/v2/ArchiveMap.json",
+  "anyOf": [
     {
       "filename": "metadata.json",
       "contentType": "application/json",
       "description": "Metadata associated with this archive including a file manifest and version information.",
       "isRequired": true,
-      "jsonSchema": "https://sage-bionetworks.github.io/mobile-client-json/schemas/v1/TaskMetadata.json"
+      "jsonSchema": "https://sage-bionetworks.github.io/mobile-client-json/schemas/v2/ArchiveMetadata.json"
     },
     {
       "filename": "taskResult.json",
       "contentType": "application/json",
       "description": "The polymorphism serialized result of running an assessment. This structure includes all the step results in the chronological order in which they were presented as well as asynchronous results that may be recorded across multiple steps.",
       "isRequired": false,
-      "jsonSchema": "https://sage-bionetworks.github.io/mobile-client-json/schemas/v1/TaskResultObject.json"
+      "jsonSchema": "https://sage-bionetworks.github.io/mobile-client-json/schemas/v1/TaskResultObject.json",
```

Comment:
yes, other than "metadata", since these aren't versioned by app, they can't change (except to subtract/add non-required properties) so I wanted to be able to support versioning for the "default" results. How I am handling that is by giving it a different filename which is why this is deprecated.